lib/src/base-styles/_nx-forms.scss

```diff
@@ -81,3 +81,20 @@
   margin: 0 0 $nx-spacing-xl 0;
   padding: 0;
 }
+
+// Inline form group
+.nx-form-row--inline-form {
+  display: flex;
+
+  .nx-form-group--button-only {
```

Comment:
What would you think of just using `nx-btn-bar` here instead of creating a new class?

Reply:
> What would you think of just using `nx-btn-bar` here instead of creating a new class?

That's much better: https://github.com/sonatype/sonatype-react-shared-components/pull/164/commits/768f86743bae3a6b994fe0f05ae2f3ea359b6d52